pom.xml

```diff
@@ -14,7 +14,7 @@
     <name>Audit Trail</name>
     <version>3.11-SNAPSHOT</version>
     <properties>
-        <jenkins.version>2.263.1</jenkins.version>
+        <jenkins.version>2.332.1</jenkins.version>
```

Comment:
Wouldn't this break compatibility with the upstream PR? https://github.com/jenkinsci/credentials-plugin/pull/295

Reply:
I could use the fallback method as seen [here](https://github.com/jenkinsci/credentials-plugin/pull/295/files/70141def75b1d0ca9b48b302a567acea07e7f7af#diff-48b8e333d1aa44a8b7cc02ad974fa46e725f73a5653da197bad628e86085cb07R59-R68). What do you think, @jglick?

Reply:
I thought about how I could make this PR backwards compatible but I think there is no point, since the credentials plugin is already at 2.340, so there will always be the dependency to that release sadly.
What would be your plans with the update schedule for this plugin? Or phrased differently: What do you think, when will we be able to merge this PR? :)

Reply:
imo you shouldn't hesitate to update the version as required.

You can always backport changes as needed, and it's not like this plugin is getting lots of releases, there hasn't been one in nearly a year. You aren't excluding any users, as they will just get the previous version.

Reply:
> You can always backport changes as needed

In case it helps: https://gist.github.com/jglick/86a30894446ed38f918050c1180483e2